test/unit/parser/parser.cpp

```diff
@@ -34,8 +33,8 @@ bool is_valid_construct(const std::string& construct) {
 }
 
 
-SCENARIO("NMODL can accept \\r as return char for one line comment", "[parser]") {
-    GIVEN("A comment defined with \\r as return char") {
+SCENARIO("NMODL can accept CR as return char for one line comment", "[parser]") {
+    GIVEN("A comment defined with CR as return char") {
```

Comment:
Maybe some mistake with the replace?

Reply:
No, \r is no more accepted in title. CR is the name of \r